rilldata/rill
# Free form ISO duration support for default_time_range

## Checklist

## Summary
#### Issue addressed: 
closes #3156

#### Details:
1. Adds support for any valid ISO duration for `default_time_range`.
2. Adds a new item to time controls when the default time range is not one of the allowed ranges. (needs design pass)

## Steps to Verify
1. Create a dashboard with a timestamp.
2. Change the `default_time_range` to something not supported like `P2W`.
4. Go to dashboard, the time range should be `Last 2 Weeks`.
5. Change the `default_time_range` to something that is supported like `P4W`.
6. Go to dashboard, time range should be `Last 4 Weeks`.
